lib/wallets/isar/models/wallet_info.dart

```diff
@@ -507,4 +507,5 @@ abstract class WalletInfoKeys {
   static const String tezosDerivationPath = "tezosDerivationPathKey";
   static const String lelantusCoinIsarRescanRequired =
       "lelantusCoinIsarRescanRequired";
+  static const String enableLelantusScanning = "enableLelantusScanning";
```

Comment:
Just for consistency I would add "Key" to the end of the string so its consistent with the others, not technically required though